src/main/java/com/wavefront/helpers/TraceTypePatternSanitizer.java

```diff
@@ -87,10 +92,23 @@ public TraceTypePattern convert(TraceTypePattern value) {
     }
     if (value.spansDistributions.isEmpty()) {
       value.spansDistributions.addAll(DEFAULT_SPANS_DISTRIBUTIONS);
-      LOGGER.warning("spansDistributions were not provided. Defaulted to " +
+      LOGGER.warning(value.traceTypeName + ": spansDistributions were not provided. Defaulted to " +
           value.spansDistributions.toString());
     }
 
+    // errorConditions
+    if (value.errorConditions != null) {
+      if (value.errorConditions.removeIf(condition -> condition.errorRate < 1 || condition.errorRate > 100)) {
+        LOGGER.warning(value.traceTypeName + ": Some of the errorConditions were removed " +
+            "because the errorRate was not in the [1..100] range!");
+      }
+      if (value.errorConditions.isEmpty()) {
+        value.errorConditions = null;
+        LOGGER.warning(value.traceTypeName + ": Empty errorConditions block was completely " +
```

Comment:
for me, it's a info, not a warning.

Reply:
Maybe the user sets empty errorConditions block and assumes that traces will be without errors, but actually, in this case, the common errorRate will be applied. For this reason, I want to attract the user's attention.